docassemble/AssemblyLine/al_general.py

```diff
@@ -259,7 +260,50 @@ def gender_fields(self, show_help=False):
         },
         self_described_input
       ]
-              
+      
+  @property
+  def gender_male(self):
+    """Provide True/False for 'male' gender to assist with checkbox filling
+    in PDFs with "skip undefined" turned on."""
+    return self.gender == 'male'
+  
+  @property
+  def gender_female(self):
+    """Provide True/False for 'female' gender to assist with checkbox filling
+    in PDFs with "skip undefined" turned on."""
+    return self.gender == 'female'
+  
+  @property
+  def gender_other(self):
+    """Provide True/False for 'other' gender to assist with checkbox filling
+    in PDFs with "skip undefined" turned on for forms without more inclusive options."""
+    return ((self.gender != 'male') and (self.gender != 'female'))
+  
```

Comment:
Is `def gender_other(self)` still needed since line 752 is deleted?

Reply:
yes, it is for non-inclusive forms with just 3 checkboxes.